app/assets/stylesheets/sufia/_file-listing.scss

```diff
@@ -112,6 +115,11 @@ h4 .small {
 .sorts-dash, .sorts, .expandable_new {
   cursor: pointer;
 }
+
+.batch-info .sort-toggle {
```

Comment:
I'm guessing this addresses the visual gap between the sort controls and the file listing?